smartMeter/ngsild-payloads/smarMeter.json

```diff
@@ -0,0 +1,64 @@
+{
+    "id": "urn:ngsi-ld:Device:smartMeter01",
+    "type": "Device",
+    "name": {
+        "type": "Property",
+        "value": "smartMeter0121215"
+    },
+    "serialNumber": {
+        "type": "Property",
+        "value": "X214356"
+    },
+    "dateInstalled": {
```

Comment:
this raises (again) the question of the terminology differences between NGSIv2 and NGSI-LD.

`dateInstalled` is NGSIv2 terminology, in NGSI-LD, we would rather call it `installedAt`.

Reply:
yes